sailthru/requirements/base.txt

```diff
@@ -1,2 +1,3 @@
+nose==1.3.7
```

Comment:
@clintonb there is already a ticket to enable Travis on this repo https://openedx.atlassian.net/browse/ECOM-6471